sdk/eventhubs/azure-messaging-eventhubs/test/perf/inc/azure/messaging/eventhubs/test/eventhubs_batch_perf_test.hpp

```diff
@@ -183,6 +183,8 @@ namespace Azure { namespace Messaging { namespace EventHubs { namespace PerfTest
 
       auto events = partitionClient.ReceiveEvents(m_batchSize, context);
       total += events.size();
+      std::cout << "[r: " << round << "/" << m_rounds << "p: " << m_partitionId
+                << "] Received: " << total << " messages" << std::endl;
```

Comment:
This change confuses me. Did something get merged strangely?

Reply:
yeah i was getitng an issue with unused variables

Reply:
Ok. That seems reasonable.